kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml

```diff
@@ -0,0 +1,32 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: bu-cs-jupyter-book
+  labels:
+    component.opendatahub.io/name: jupyterhub
+    opendatahub.io/component: 'true'
+    opendatahub.io/notebook-image: 'true'
+  annotations:
+    opendatahub.io/notebook-image-name:
+      "BU CS Jupyter Book"
+    opendatahub.io/notebook-image-url:
+      "https://github.com/aicoe-aiops/openshift-anomaly-detection"
+    opendatahub.io/notebook-image-name:
+      "OpenShift Anomaly Detection Notebook Image"
+    opendatahub.io/notebook-image-desc:
+      "Image with jupyter notebooks for anomaly detection and diagnosis discovery in OCP clusters"
```

Comment:
Update this with the description of your image

Reply:
Thanks I have updated the stanzas